examples/JointCases/fixed_joint.py

```diff
@@ -75,7 +81,13 @@ class FixedJointSimulator(
 # Connect rod 1 and rod 2
 fixed_joint_sim.connect(
     first_rod=rod1, second_rod=rod2, first_connect_idx=-1, second_connect_idx=0
-).using(FixedJoint, k=1e5, nu=0, kt=5e3)
+).using(
+    FixedJoint,
+    k=1e5,
+    nu=0e0,
+    kt=1e1,
```

Comment:
@armantekinalp I actually badly tuned this spring constant for the torsion before in the previous PR. The example simulation was very unstable and fragile as it would disconverge often. 

As this spring is now directly acting on the rotation instead some position difference (as changed in previous PR), a lower spring constant seems to be suitable.

Reply:
okay thanks for clarifying.